src/lib/y2partitioner/widgets/fstab_options.rb

```diff
@@ -281,6 +373,20 @@ def selected_mount_by
       def value
         Yast::UI.QueryWidget(Id(:mt_group), :Value)
       end
+
+    private
+
+      def select_default_mount_by
+        # Mount point has always a value for mount_by. By default, MountPoint#mount_by
+        # backwards to the Storage#default_mount_by value, taking into account
```

Comment:
I guess you mean "fallbacks" instead of "backwards". But even though, I still don't think the explanation about `#default_mount_by` fits here. This gets the value from the current mount_point, where that value comes from is not much relevant here.

Reply:
Ok, I will simply remove the comment.